opencga-core/src/main/java/org/opencb/opencga/core/models/study/CatalogServiceConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package org.opencb.opencga.core.models.study;
+
+public class CatalogServiceConfiguration {
+
+    private boolean active;
```

Comment:
Please, remember to add `@DataField()` annotations to any new catalog object with meaningfull descriptions